### PR TITLE
K8 engine: separate configmap for logback conf & modelConfig in secret

### PR DESCRIPTION
**Pull request by gadomsky**
Hi,
Please add changelog entry in docs/Changelog.md.
After that update description of this PR.
Thanks.

### CODE REVIEW

**Review by arkadius — APPROVED** [review]

File: engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala [hunk 1]
```diff
@@ -13,16 +13,24 @@ import skuber.LabelSelector.IsEqualRequirement
 import skuber.apps.v1.Deployment
 import skuber.{Container, EnvVar, HTTPGetAction, LabelSelector, Pod, Probe, Volume}
 
+case class MountableResources(
```
Comment (dswiecki) [inline engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala:16]: this formatting looks wired, more often we do not make empty line after case class name

File: engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala [hunk 2]
```diff
@@ -13,16 +13,24 @@ import skuber.LabelSelector.IsEqualRequirement
...
+                              commonConfigConfigMap: String,
```
Comment (dswiecki) [inline engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala:17]: is commonConfigConfigMap a whole config map ?

File: engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala [hunk 3]
```diff
@@ -94,15 +94,23 @@ class K8sDeploymentManager(modelData: BaseModelData, config: K8sDeploymentManage
...
-      //we append hash to configMap name so we can guarantee pods will be restarted.
-      //They *probably* will restart anyway, as scenario version is in label, but e.g. if only model config is changed?
-      deployment <- k8sUtils.createOrUpdate(k8s, deploymentPreparer.prepare(processVersion, configMap.name, scalingOptions.replicasCount))
+      configMap <- k8sUtils.createOrUpdate(configMapForData(processVersion, canonicalProcess, config.nussknackerInstanceName)(Map(
+        "scenario.json" -> canonicalProcess.asJson.noSpaces,
+        "deploymentConfig.conf" -> ConfigFactory.empty().withValue("tasksCount", fromAnyRef(scalingOptions.noOfTasksInReplica)).root().render()
+      )))
+      loggingConfigMap <- k8sUtils.createOrUpdate(configMapForData(processVersion, canonicalProcess, config.nussknackerInstanceName)(Map("logback.xml" -> logbackConfig), additionalLabels = Map(resourceTypeLabel -> "logging-conf")))
+      secret <- k8sUtils.createOrUpdate(secretForData(processVersion, canonicalProcess, config.nussknackerInstanceName)(Map("modelConfig.conf" -> serializedModelConfig)))
```
Comment (dswiecki) [inline engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala:102]: maybe let's leave a comment why we put in secret

File: engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala (hunk 2 above)
Comment (gadomsky) [reply engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala]: it contains deployment.confg and scenario.json

**Pull request merged**